### PR TITLE
Register Subdomain: Thebidouilleur

### PR DESCRIPTION
**Pull request by qjoly**
Added the domain: Thebidouilleur

### CODE REVIEW

File: domains/Thebidouilleur.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "QJoly",
+        "email": "quentinjoly@thoughtless.eu"
+      },
+
+      "record": {
+        "A": ["thoughtless.eu"]
```
Comment (WilliamDavidHarrison) [inline domains/Thebidouilleur.json:9]: Please use a CNAME for this.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/Thebidouilleur.json (hunk 1 above)
Comment (QJoly) [reply domains/Thebidouilleur.json]: Changed by my IP

**Comment by QJoly** [comment]
Updated ! 
Thanks in advance :)

File: domains/Thebidouilleur.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "QJoly",
+        "email": "quentinjoly@thoughtless.eu"
+      },
+
+      "record": {
+        "A": ["5.39.75.213"]
```
Comment (WilliamDavidHarrison) [inline domains/Thebidouilleur.json:9]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/Thebidouilleur.json (hunk 2 above)
Comment (QJoly) [reply domains/Thebidouilleur.json:9]: I'm using a reverse proxy, by default : it's just a 404. 
If you curl this IP with a custom header, you will have a result :) 

```bashrc
 curl https://5.39.75.213/ -H "Host:git.thoughtless.eu" -k
 ```

**Review by wdhdev — DISMISSED** [review]
Uses reverse proxy. Retrieve content using:
```curl https://5.39.75.213/ -H "Host:git.thoughtless.eu" -k```

**Review by wdhdev — APPROVED** [review]
Uses reverse proxy. Retrieve content using:
```curl https://5.39.75.213/ -H "Host:git.thoughtless.eu" -k```

**Review by andrewstech — APPROVED** [review]

**Pull request merged**